ecosystem/sep-0008.md

```diff
@@ -225,17 +225,56 @@ Name | Type | Description
 `status` | string | `"action_required"`
 `message` | string | A human readable string containing information regarding the action required.
 `action_url` | string | A URL that allows the user to complete the actions required to have the transaction approved.
+`action_method` | string | (Optional) `GET` or `POST`, indicating the type of request that should be made to the `action_url`. If not provided, `GET` is assumed.
+`action_fields` | string[] | (Optional) An array of additional fields defined by [SEP-9 Standard KYC / AML fields] that the client may optionally provide to the approval service when sending the request to the `action_url`.
 
 ###### Example
 
 ```json
 {
   "status": "action_required",
   "message": "Please sign the terms of service via the provided URL.",
-  "action_url": "https://..."
+  "action_url": "https://...",
+  "action_method": "POST",
+  "action_fields": ["email_address", "mobile_number"]
 }
 ```
 
+###### Action URL
+If the `action_method` is provided the client uses the type of request indicated when sending a request to the `action_url`. If `action_fields` are to be provided and the fields contain sensitive or personal information the `POST` case is preferred to ensure sensitive information is not transmitted in the URL.
+
+- In the case that `action_method` is `GET` any fields requested may be passed as query parameters in the `action_url`.
+
+- In the case that `action_method` is `POST` any fields requested may be passed as JSON in the request body, using content type `application/json`. The server should respond to the `POST` with a HTTP response with status code `200`, content-type `application/json` and a body containing the next URL that the user should be taken to.
+
+   The next URL in the POST case is necessary to support products that are websites and products that are mobile applications. Products that run entirely in the browser can redirect the user to the next URL and the redirect prevents odd user experiences where a user refreshes a page and is prompted to re-post the data. Products that are mobile applications cannot open a system browser with a POST request. They can make the POST request internally, extract the redirect in the response, and open a system browser with the next URL.
+
+   Request Parameters:
+
+   Any fields defined by [SEP-9 Standard KYC / AML fields].
+
+   Request Example:
+
+   ```json
+   {
+     "email_address": "user@example.com"
+   }
+   ```
```

Comment:
Note: This portion discussing the request format does not discuss how to send SEP-9 fields that have sub-fields. I'm intentionally leaving that undefined until I get an understanding of how SEP-12 works, so that I can make SEP-8 consistent with SEP-12. Related discussion: https://github.com/stellar/stellar-protocol/issues/832